### PR TITLE
Utilize OTP A11y score

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
Replaces the `getAccessibilityScoreForItinerary` function with the `accessibilityScore` that is returned in the OTP itinerary.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
So much better!

File: lib/util/accessibility-routing.js
Comment (binh-dam-ibigroup) [inline lib/util/accessibility-routing.js:4]: Because we are looking for an accessiblity score for all legs, we should rename this function to `itineraryHasAccessibilityScoresOnAllLegs` or `hasAccessibilityScoresOnAllLegs`.

File: lib/util/accessibility-routing.js
Comment (binh-dam-ibigroup) [inline lib/util/accessibility-routing.js:5]: Can the accessibility score of a leg be zero? if so, the whole itinerary score will NOT be displayed, and in such cases it should be displayed.

**Review by binh-dam-ibigroup — COMMENTED** [review]
See my two comments at first glance.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Please advise on cases where a leg accessibility score is zero.

**Comment by amy-corson-ibigroup** [comment]
I think the accessibility score would only be zero if one of the legs of the trip is absolutely confirmed to be inaccessible to wheelchair users. Wouldn't that be a trip we want to exclude?

**Comment by binh-dam-ibigroup** [comment]
> I think the accessibility score would only be zero if one of the legs of the trip is absolutely confirmed to be inaccessible to wheelchair users. Wouldn't that be a trip we want to exclude?

Do you mean OTP would not return such an itinerary?

File: lib/util/accessibility-routing.js
Comment (amy-corson-ibigroup) [reply lib/util/accessibility-routing.js:4]: Since the previous version of this function was just checking for itinerary scores on any of the legs, I repurposed it to check for the itinerary score itself! Which should make the name of this function still make sense but let me know if you disagree.

File: lib/util/accessibility-routing.js
Comment (amy-corson-ibigroup) [reply lib/util/accessibility-routing.js]: Yes, understand what you mean now! Thank you, fixed in [92694f2](https://github.com/opentripplanner/otp-react-redux/pull/1092/commits/92694f2f351b69949b3ed95a9f9607c432f3fff5)

File: lib/util/accessibility-routing.js
Comment (binh-dam-ibigroup) [reply lib/util/accessibility-routing.js:4]: Sounds good. But it should be singular (`itineraryHasAccessibilityScore`) since you are only checking one score!

File: lib/util/accessibility-routing.js
Comment (binh-dam-ibigroup) [inline lib/util/accessibility-routing.js:6]: Please add a comment regarding the -1 value.

**Review by binh-dam-ibigroup — APPROVED** [review]
Approving after internal discussion, and subject to the tweaks indicated. Sorry for holding up this work.

File: lib/util/accessibility-routing.js
Comment (amy-corson-ibigroup) [reply lib/util/accessibility-routing.js]: Good point, changed in [ab6574e](https://github.com/opentripplanner/otp-react-redux/pull/1092/commits/ab6574ee9fe335fddcb9706508ebf9933ebc74a2)!

File: lib/util/accessibility-routing.js
Comment (amy-corson-ibigroup) [reply lib/util/accessibility-routing.js:7]: Added in [ab6574e](https://github.com/opentripplanner/otp-react-redux/pull/1092/commits/ab6574ee9fe335fddcb9706508ebf9933ebc74a2)!

**Comment by amy-corson-ibigroup** [comment]
Not at all! These were helpful changes!

File: lib/util/accessibility-routing.js
Comment (binh-dam-ibigroup) [reply lib/util/accessibility-routing.js:7]: The comment talks about zero which is good, how about the -1?

Comment (amy-corson-ibigroup) [reply lib/util/accessibility-routing.js:7]: I missed this comment before I merged, I'm sorry! I will sneak an updated comment into another otp-rr pr

**Pull request merged**
